addon/webextension/.eslintrc.json

```diff
@@ -1,13 +0,0 @@
-{
-    "env": {
-        "browser": true,
-        "es6": true,
-        "webextensions": true
-    },
-    "extends": [
-      "eslint:recommended"
-    ],
-    "rules": {
-      "no-console": "warn"
-    }
-}
```

Comment:
For some reason, this file was preventing ESLint from loading the parent config. Not sure why.

Reply:
I think this can be resolved by moving the ESLint config file up a directory into /addon/:

```
"use strict";

module.exports = {
  env: {
    browser: true,
    es6: true,
    webextensions: true,
  },
};
```

I guess maybe ESLint only looks for config files up the tree as long as there are no gaps.